### PR TITLE
Alps 577 hotspot sprite

### PR DESCRIPTION
**Pull request by rroux-gpsw**
Hotspot sprite material implementation.
Hotspot video material implementation.
You can test the sample hotspots-sprite.
A sample for video material is coming tomorrow.

### CODE REVIEW

**Comment by sw-team-release-gpsw** [comment]
[CLA assistant check](https://cla.gopro.com/sign) <br/>All committers have signed the CLA.

File: src/display/sprite/SpriteAnimationManager.js [hunk 1]
```diff
@@ -173,11 +198,21 @@ FORGE.SpriteAnimationManager.prototype.play = function(animation, loop, index)
...
+        this._pending.push({
+            method: "pause",
+            args: [index]
+        });
 
-    if(anim !== null)
+        return;
+    }
+
+    //index = (typeof index === "number" && isNaN(index) === false) ? index : 0;
```
Comment (aberthet-gpsw) [inline src/display/sprite/SpriteAnimationManager.js]: Remove this if unused

File: src/hotspots/HotspotMaterial.js [hunk 2]
```diff
@@ -266,16 +292,117 @@ FORGE.HotspotMaterial.prototype._createTextureFromImage = function(image)
 {
     this._displayObject = image;
 
-    this.setTextureFrame(image.frame);
+    this._texture = new THREE.Texture();
+    this._texture.generateMipmaps = false;
+    this._texture.minFilter = THREE.LinearFilter;
```
Comment (aberthet-gpsw) [inline src/hotspots/HotspotMaterial.js]: We may have a problem here, as hotspots are smoothed again on desktop, but are now really aliased and ugly when in WebVR (seen on GearVR). Either we change this depending on the type of view (normal or vr), either we find a compromise between the two.

File: reference/display/sprite-animation.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (aberthet-gpsw) [inline reference/display/sprite-animation.json]: Add those files in `tools/reference/pages.json`

File: src/display/sprite/SpriteAnimationManager.js (hunk 1 above)
Comment (rroux-gpsw) [inline src/display/sprite/SpriteAnimationManager.js]: Removed

File: src/hotspots/HotspotMaterial.js (hunk 2 above)
Comment (rroux-gpsw) [inline src/hotspots/HotspotMaterial.js]: Ok, will find a way to remove aliasing on VR on another pull request if it's ok for the team! This needs a lot of testing on different screen / texture sizes.

File: reference/display/sprite-animation.json (hunk 3 above)
Comment (rroux-gpsw) [inline reference/display/sprite-animation.json]: Done. Thanks!

**Pull request merged**